docs/api/data-models/service-lookup-request.md

```diff
@@ -3,12 +3,12 @@
 Field | Type | Mandatory | Description
 --- | --- | --- | ---
 instanceIds | List<[ServiceInstanceID](../primitives.md#serviceinstanceid)> | no | Requester is looking for service instances with any of the specified identifiers. Mandatory if no providerNames nor serviceDefinitionNames are specified.
```

Comment:
Ususally we put `yes/no` into Mandatory field if both can be possible, which I think is a good practice.